introduction/what-is.md

```diff
@@ -0,0 +1,45 @@
+This file holds the text to accompany the [What is InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321608) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What is InnerSource?
+
+The basic idea of InnerSource is this: If a consuming team can't get what it needs by submitting a feature request then it may submit a pull request instead.
```

Comment:
Proposal: `s/a consuming team/a team using InnerSource software`. Or maybe even use _team A_ and _team B_ to avoid misunderstandings. Here's an attempt:

> Let's say team A is using software produced by team B. Team A submitted a feature request to team B but team B isn't able to implement that feature in time for team A. In an InnerSource setting, team A would then implement the feature itself and submit a Pull Request to team B, which would then partner to review and accept the pull request and thus add the feature to their software. In this example, we call team A the **Guest** team and team B the **Host** team.

Reply:
Updated.  Have a look.